pkg/kubectl/cmd/util/editor/editoptions.go

```diff
@@ -280,6 +282,8 @@ func (o *EditOptions) Run() error {
 			switch o.EditMode {
 			case NormalEditMode:
 				err = o.visitToPatch(infos, updatedVisitor, &results)
+			case ApplyEditMode:
+				err = o.applyEditPatch(infos, updatedInfos)
```

Comment:
I'm surprised this isn't following the same pattern of passing in the `updatedVisitor` and visiting the results